yaptide/routes/celery_routes.py

```diff
@@ -158,21 +162,16 @@ def delete(user: UserModel):
                                     })
 
         tasks = fetch_celery_tasks_by_sim_id(sim_id=simulation.id)
-        celery_ids = [
-            task.celery_id for task in tasks
-            if task.task_state in [EntityState.PENDING.value, EntityState.RUNNING.value, EntityState.UNKNOWN.value]
-        ]
-
-        # The merge_id is canceled first because merge task starts after run simulation tasks are finished/canceled.
-        # We don't want it to run accidentally.
-        celery_app.control.revoke(simulation.merge_id, terminate=True, signal="SIGINT")
-        celery_app.control.revoke(celery_ids, terminate=True, signal="SIGINT")
-        update_simulation_state(simulation=simulation, update_dict={"job_state": EntityState.CANCELED.value})
-        for task in tasks:
-            if task.task_state in [EntityState.PENDING.value, EntityState.RUNNING.value]:
-                update_task_state(task=task, update_dict={"task_state": EntityState.CANCELED.value})
-
-        terminate_unfinished_tasks.delay(simulation_id=simulation.id)
+        celery_ids = []
```

Comment:
what are celery_ids ?

Reply:
List of celery_id from CeleryTaskModel, residue from the old version of the code